src/L1ERC20Bridge.sol

```diff
@@ -50,4 +52,19 @@ contract L1ERC20Bridge is L1VotePool {
     nonce = nonce + 1;
     return sequence;
   }
+
+  /// @notice Receives an encoded withdrawal message from the L2
+  /// @param encodedMsg The encoded message from the L2 with the withdrawal information.
+  function receiveEncodedWithdrawalMsg(bytes memory encodedMsg) public {
```

Comment:
This function can now receive two types of messages:

1. Vote messages
2. Withdrawal messages

How does Wormhole know which function signature to call with each type of bridge message? Should we also change the name of the vote receiving method to make it clear what it's for?

Reply:
With the specialized relayer we would write the relayer call. So, the method name could be anything as long as we coded the specialized relayer properly. In a future pr when we switch to the generic relayer the method name is sent to the generic relayer.

Reply:
We should definitely update the name to make it more clear.

Reply:
So, `receiveEncodedMsg` is in the `WormholeReceiver` interface. Renaming should be handled in the generic relayer pr.